src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.companieshouse.controller;
+
+import io.swagger.v3.core.util.Json;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.headers.Header;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.service.AppealService;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/companies")
+public class AppealController {
+
+    public final AppealService appealService;
+
+    @Operation(summary = "Create a new appeal", tags = "Appeal")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "201", description = "Appeal resource created", headers = {
+            @Header(name = "location")
+        }),
+        @ApiResponse(responseCode = "401", description = "Unauthorised request"),
```

Comment:
In tests 400 response is simulated but doc does not mention it.

Reply:
by default if the required parameters are not sent, an 400 response code is returned. I'll update the docs to reflect this, however we did want to return a status code 422 instead